tools/structure/src/utils.py

```diff
@@ -0,0 +1,31 @@
+from typing import Any
+
+
+def json_to_markdown(data: Any, level: int = 0, parent_key: str = "") -> str:
```

Comment:
This looks like a common piece of code that can be re-used? Might want to consider moving this for later. Also writing such logical pieces - Has this been practically re-usable for us? Or have we been spending quite some effort in doing such things? We may have to re-consider our strategy if we think we are spending a lot of tie in writing such parsing logic.

Reply:
@gaya3-zipstack I agree in moving such common code logic to a relevant lib. Will move this to the SDK later and remove this from here when we encounter a scenario where we wish to use this.

From what I've seen, the common function itself gets duplicated across components sometimes and we end up refactoring it later.
This particular function can be useful to render any JSON for any other tool too. Currently I don't see a usecase for it yet.

Also it was easy to obtain such a generic parsing logic from an LLM :)